.env.example

```
@@ -0,0 +1,10 @@
+# 필수
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+POSTGRES_DB=
+
+# 선택 (기본값 사용 시 생략 가능)
+# POSTGRES_PORT=5432
+# REDIS_PORT=6379
+# KAFKA_PORT=9092
+# KAFKA_UI_PORT=8989
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)